pkg/templates/templates/credentials/create/credential-set.yaml

```diff
@@ -0,0 +1,13 @@
+schemaType: CredentialSet
+schemaVersion: 1.0.0
+name: NAME
+# Omit the namespace to define the credential set in the current current namespace
+namespace: NAMESPACE
+# Optionally define labels which can be used for filtering. Labels must be string values.
+labels:
+  LABEL: "LABEL_VALUE"
+credentials:
+  - name: CREDENTIAL_NAME
+    source:
+      # Replace the value field below with any of the following: secret, env, path, command or value
+      env: CREDENTIAL_VALUE
```

Comment:
What do you think about having multiple credentials included in the template, each demonstrating a popular source: value, env, secret perhaps? Would that be helpful?

Reply:
This is a really great input which will definitely be more helpful to the user to provide more examples that encompass all of the possibilities of credentials. Thank you lots for the much needed reminder! 😄